cli.js

```diff
@@ -358,12 +358,17 @@ process.on('uncaughtException', function(err) {
       var sfxBundle = true;
 
     case 'bundle':
-      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime']);
+      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime', '--inline-source-maps']);
       if (options.yes)
         ui.useDefaults();
       options.sourceMaps = !options['skip-source-maps'];
       options.lowResSourceMaps = !options['hires-source-maps'];
       options.mangle = !options['no-mangle'];
+      
+      if (options['inline-source-maps']){
+        options.sourceMapContents = 'inline';
```

Comment:
That's the complete stack trace, yes.
I have no idea where the error is coming from.

I don't do anything fancy to bundle css, its done in a separated process using lessc which doesn't interact with this. It's probably due to one of my deps.

I think we need to differentiate the line you pointed and this [one](https://github.com/systemjs/builder/blob/master/lib/output.js#L69).

The way I understand what's going on, my version of the code embeds the source in the sourcemap, while yours embeds the sourcemap in the source, right ?

all you should need to reproduce is [here](https://gist.github.com/johnraz/03fae8f6fd602f480fe1)